README.md

```diff
@@ -1 +1,120 @@
-# My-Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <h3><b>My Portfolio</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 [My Portfolio] <a name="about-project"></a>
+
+**[My Portfolio]** is a is a brief sample of the work done in my career as a software developer.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Html & Css.
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive]**
+- **[Flex]**
+- **[Hover Reactions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
```

Comment:
- Please remove all lines from your readme that start with `>`. These lines are comments meant to guide you the project owner on how to fill the readme and should not be present for your users to see